build/media-suite_helper.sh

```diff
@@ -1378,16 +1378,36 @@ log() {
 }
 
 create_build_dir() {
-    local build_dir="build${1:+-$1}-$bits"
-    if [[ ! -f "$(get_first_subdir)/do_not_clean" ]]; then
-        if [[ "$(basename "$(pwd)")" == "$build_dir" ]]; then
-            rm -rf ./* && cd_safe ".."
-        elif [[ -d $build_dir ]] && ! rm -rf ./"$build_dir"; then
-            cd_safe "$build_dir" && rm -rf ./* && cd_safe ".."
-        fi
+    local print_build_dir=false nocd=${nocd:-false} norm=false build_root repo_dir build_dir getoptopt OPTARG OPTIND
+    build_root="$LOCALBUILDDIR/$(get_first_subdir)"
+    while getopts ":pcrC:" getoptopt; do
+        case $getoptopt in
+        p) print_build_dir=true ;;
+        c) nocd=true ;;
+        r) norm=true ;;
```

Comment:
Is this option supposed to be doing something?

Reply:
I feel like it was supposed to be an alternate to `do_not_clean` but for use with printing a build dir without cleaning it, but I guess I forgot to actually implement it

Reply:
Added in line 1406